src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/SearchJqlWithoutStarAndReporterAdminAction.kt

```diff
@@ -0,0 +1,37 @@
+package com.atlassian.performance.tools.jiraactions.api.action
+
+import com.atlassian.performance.tools.jiraactions.api.SEARCH_WITH_JQL_WITHOUT_WILDCARD_AND_PREVIOUS_REPORTER
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
+import com.atlassian.performance.tools.jiraactions.api.observation.SearchJqlObservation
+import com.atlassian.performance.tools.jiraactions.api.page.IssueNavigatorPage
+import java.util.function.Predicate
+import javax.json.JsonObject
+
+class SearchJqlWithoutStarAndReporterAdminAction(
```

Comment:
I don't like the name. For me it is definitely too long. Perhaps we should just name it `SearchJqlOther`? And maybe add short KDoc saying what JQLs are included?

Reply:
Ah sorry, I did not see the previous conversation.

Reply:
SearchJQLSubset ?